Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,31 +15,214 @@ public static class BalanceStats
         /// </summary>
         public static string FindPersonWithBiggestLoss(string[] peopleAndBalances)
         {
-            return "";
+            if (IsNullOrEmpty(peopleAndBalances)) return Constants.Nothing;
+            Thread.CurrentThread.CurrentCulture = Config.Culture();
+
+            var (names, money, isPlural) = ReturnNamesMoneyAndIsPlural(peopleAndBalances, "min");
+            if (money >= 0) return "N/A.";
+
+            return $"{names} lost the most money. {FixMoneyFormat(money)}.";
         }
 
         /// <summary>
         /// Return name and current money of the richest person.
         /// </summary>
         public static string FindRichestPerson(string[] peopleAndBalances)
         {
-            return "";
+            if (IsNullOrEmpty(peopleAndBalances)) return Constants.Nothing;
+            Thread.CurrentThread.CurrentCulture = Config.Culture();
+
+            var (names, money, isPlural) = ReturnNamesMoneyAndIsPlural(peopleAndBalances, "rich");
+            var dictionary = new string[]
+            {
+                (isPlural) ? "are" : "is",          // [0]
+                (isPlural) ? "people" : "person"    // [1]
+            };
+
+            return $"{names} {dictionary[0]} the richest {dictionary[1]}. {FixMoneyFormat(money)}.";
         }
 
         /// <summary>
         /// Return name and current money of the most poor person.
         /// </summary>
         public static string FindMostPoorPerson(string[] peopleAndBalances)
         {
-            return "";
+            if (IsNullOrEmpty(peopleAndBalances)) return Constants.Nothing;
+            Thread.CurrentThread.CurrentCulture = Config.Culture();
+
+            var (names, money, isPlural) = ReturnNamesMoneyAndIsPlural(peopleAndBalances, "poor");
+            var dictionary = new string[]
+            {
+                (isPlural) ? "have" : "has",        // [0]
+            };
+
+            return $"{names} {dictionary[0]} the least money. {FixMoneyFormat(money)}.";
+        }
+
+        private static (string names, decimal money, bool isPlural) ReturnNamesMoneyAndIsPlural(string[] peopleAndBalances, string option)
+        {
+            if (!IsOptionValid(option)) throw new Exception($"{option} is not a valid option for function ProcessPeopleBalance().");
+
+            var (name, money) = ProcessPeopleBalance(peopleAndBalances, option);
+            var people = new List<string>();
+            var moneyToCompare = (option == "max" || option == "rich") ? money.Max() : money.Min();
+
+            for (var i = 0; i < money.Count; i++)
+            {
+                if (money[i] == moneyToCompare) people.Add(name[i]);
+            }
+
+            var (processedNames, isPlural) = ProcessName(people);
+
+            return (processedNames, moneyToCompare, isPlural);
+        }
+
+        private static (List<string> name, List<decimal>) ProcessPeopleBalance(string[] peopleAndBalances, string option)
```

Comment:
Avoid using list, avoid using class for the first chapter. Pretend you don't know about the list yet. Also, don't return tuples or deconstruction. Return a single value and call different functions multiple times. For chapter 2 you will learn objects, then you can prefer that. This is not very readable.